.github/workflows/ci_cd.yml

```diff
@@ -13,7 +13,7 @@ on:
 
 env:
   MAIN_PYTHON_VERSION: '3.9'
-  ANSYS_VERSION: '231'
+  CONTAINER_TAG: 'fix_temp'
```

Comment:
should this be latest?

Reply:
Yes. This was just a temporary state because I wanted a container that does not get overwritten by dpf_composites builds for develop